app/containers/Root.js

```diff
@@ -40,39 +38,19 @@ class Root extends React.Component {
 
     initTheme: PropTypes.func.isRequired,
     isMounted: PropTypes.bool.isRequired,
-    setMounted: PropTypes.func.isRequired,
-    setLoading: PropTypes.func.isRequired
-  }
-
-  state = {
-    timer: null
+    setMounted: PropTypes.func.isRequired
   }
-
   /**
    * // Show the loading bold briefly before showing the user the app.
    */
   componentDidMount() {
-    const { initTheme, isLoading, isMounted, setLoading, setMounted, theme } = this.props
+    const { initTheme, isMounted, setMounted } = this.props
 
-    // If this is the first time the app has mounted, initialise things.
+    // If this is the first time the app has mounted, initialize things.
     if (!isMounted) {
       setMounted(true)
       initTheme()
     }
-
-    // Hide the loading screen after a set time.
-    if (isLoading || !theme) {
-      const timer = setTimeout(() => setLoading(false), SPLASH_SCREEN_TIME)
```

Comment:
Without this, when you start up the app that has no active/open wallet the loading graphic flickers onscreen for a split second before the user is shown the home screen.

I think we should try to keep the minimum splash screen visibility time if possible.

Ideally, the loading graphic should never show for less than a second, as it doesn't look good when it flickers on and off incredibly quickly.